src/Components/CreateImageWizard/utilities/requestMapper.ts

```diff
@@ -172,9 +172,22 @@ function commonRequestToState(
     (image) => image.image_type === 'azure'
   );
 
-  const snapshot_date =
+  const snapshotDateFromRequest =
     request.image_requests.find((image) => !!image.snapshot_date)
       ?.snapshot_date || '';
+  let snapshot_date = '';
```

Comment:
maybe it is make sense to add comments with explanation about the format itslef?

Reply:
Completely missed this one, sorry! I've added a comment with explanation about the date formats.